components/src/core/Hero/Hero.tsx

```diff
@@ -97,9 +118,13 @@ export const Hero = (props: HeroProps): JSX.Element => {
                 {!props.children && value && <ChannelValue value={value} units={units} icon={valueIcon} />}
                 {props.children}
             </span>
-            <Typography variant={'subtitle1'} color={'inherit'} className={classes.label}>
-                {label}
-            </Typography>
+            {showTooltipOnHover ? (
+                <Tooltip title={label} classes={tooltipClasses} arrow>
+                    {labelElement}
+                </Tooltip>
+            ) : (
+                labelElement
+            )}
```

Comment:
Is this the style we want for tooltips? We should probably discuss how we want our standard tooltips to look.